chanzuckerberg/idseq-web
# Change host genome default to ERCC only

# Description

This changes the model definition to make the default files of new host genomes to be ERCC only, in preparation for plain text host genomes. 

# Notes

* I took the opportunity to add some validation and clean up of the model. 

# Tests

* run migration, see all existing records validate
* run new rspec test
